src/vm/dwbucketmanager.hpp

```diff
@@ -16,18 +16,10 @@
 #ifndef DWBUCKETMANAGER_HPP
 #define DWBUCKETMANAGER_HPP
 
-#ifdef FEATURE_WINDOWSPHONE
-#include "corhost.h"
-#endif
-
 // this will be used as an index into g_WerEventTraits
 enum WatsonBucketType
 {
     CLR20r3 = 0,
-    MoCrash,
-#ifdef FEATURE_WINDOWSPHONE
```

Comment:
@leculver I am deleting the WindowsPhone-specific Watson bucketing. Let me know if you see a problem with it.

Reply:
@jkotas I'm super happy with the change.  My only question is whether we should preserve the WatsonBucketType 1 and 2 as "reserved for historical reasons", so that future enums don't collide with them...but I don't think this is used by anything that matters?

Reply:
Right, it is not used for anything that matters. I have actually considered deleting it - I have not done it just because of the code would make less sense without further refactoring.